hive-metastore/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -177,20 +179,17 @@ public boolean dropTable(TableIdentifier identifier, boolean purge) {
     }
 
     try {
-      clients.run(
-          client -> {
-            client.dropTable(
-                database,
-                identifier.name(),
-                false /* do not delete data */,
-                false /* throw NoSuchObjectException if the table doesn't exist */);
-            return null;
-          });
-
+      clients.run(client -> {
+        client.dropTable(database, identifier.name(),
+            false /* do not delete data */,
+            false /* throw NoSuchObjectException if the table doesn't exist */);
+        return null;
+      });
```

Comment:
This seems to be a purely formatting change. It is unnecessary.

Reply:
Thanks @wypoon for your comment!
Yes, I meanwhile I also learnt that this PR was a bit naive and it involves a much higher level topic about Catalogs being responsible to guarantee that dropping table location is safe.
I think I'll close this PR, then.